src/resolve/resolved-dns-query.c

```diff
@@ -585,6 +586,17 @@ void dns_query_complete(DnsQuery *q, DnsTransactionState state) {
 
         q->state = state;
 
+        if (state == DNS_TRANSACTION_SUCCESS && set_size(q->manager->varlink_subscription) > 0) {
+                DnsQuestion *question = q->request_packet ? q->request_packet->question :
+                                                            NULL;
```

Comment:
Not a matter, but it is not necessary to wrap this line.

Reply:
I unwrapped the line now